app/controllers/admin/users_controller.rb

```diff
@@ -6,9 +6,10 @@ def index
   end
 
   def show
-    @merchant = User.find(params[:id])
+    @user = User.find(params[:id])
+    @user_role = @user.role
```

Comment:
Very nice work here, I was wondering when we were goin to stop using just merchants 👍